src/app/core/abstractions/rundown.service.ts

```diff
@@ -4,6 +4,7 @@ import { Rundown } from '../models/rundown'
 export abstract class RundownService {
   public abstract fetchRundown(rundownId: string): Observable<Rundown>
   public abstract activate(rundownId: string): Observable<void>
+  public abstract rehearsal(rundownId: string): Observable<void>
```

Comment:
This is a method name so it should use the verb `rehearse`

Reply:
Fixed